feather/server/src/logging.rs

```diff
@@ -16,9 +18,18 @@ pub fn init(level: LevelFilter) {
             } else {
                 record.module_path().unwrap_or_default()
             };
+
+            let datetime: OffsetDateTime = match OffsetDateTime::now_local() {
+                Ok(x) => x,
+                Err(_) => OffsetDateTime::now_utc(),
+            };
             out.finish(format_args!(
                 "{} {:<5} [{}] {}",
-                chrono::Local::now().format("%Y-%m-%d %H:%M:%S,%3f"),
+                datetime
+                    .format(format_description!(
+                        "[year]-[month]-[day] [hour]:[minute]:[second],[subsecond digits:3]"
```

Comment:
Can we not just use multiple `{}` and get the year, month, day, hour, minute and second manually? Would it not be faster and safer than using format description?

Reply:
what's your basis for claiming that would be faster and safer?

Reply:
I don't think its safer, but the format has to be recompiled every time the function is called

Reply:
I assume its faster to only allocate one String instead of two which gets merged... other than that its just an assumption.

Reply:
Solved by @Gator